bizdemo/easy_note/cmd/api/main.go

```diff
@@ -77,5 +112,19 @@ func main() {
 	note1.POST("", handlers.CreateNote)
 	note1.PUT("/:note_id", handlers.UpdateNote)
 	note1.DELETE("/:note_id", handlers.DeleteNote)
+
+	r.NoRoute(func(ctx context.Context, c *app.RequestContext) {
+		c.JSON(http.StatusNotFound, map[string]interface{}{
+			"code":    errno.NoRouteErrCode,
+			"message": errno.NoRouteErr.ErrMsg,
+			"data":    nil,
+		})
+	})
+	r.NoMethod(func(ctx context.Context, c *app.RequestContext) {
```

Comment:
refer to https://www.cloudwego.io/zh/docs/hertz/tutorials/basic-feature/route/#noroute-%E4%B8%8E-nomethod-%E4%BD%BF%E7%94%A8

Reply:
i got it, thx